app/controllers/staff/proposal_reviews_controller.rb

```diff
@@ -35,7 +35,12 @@ def show
   end
 
   def update
-    authorize @proposal, :review?
+    if params[:program_mode] = true
```

Comment:
Thanks! I missed that ==. As far as program_mode?, that is what I initially hoped to use, but it doesn't work here as written since it's going through the reviews controller. I'll take a deeper look at restructuring how this is working.